package.json

```diff
@@ -46,7 +46,7 @@
     "copy-webpack-plugin": "4.0.1",
     "coveralls": "^2.11.15",
     "css-loader": "^0.26.0",
-    "d3": "^4.4.1",
+    "d3": "^4.9.1",
```

Comment:
https://github.com/c3js/c3/issues/1648 is the c3 issue on d3 4 compatibility - short answer - waiting for c3 1.0.0